src/containers/Header/Search.tsx

```diff
@@ -1,13 +1,16 @@
-import { KeyboardEventHandler, useContext } from 'react'
+import { KeyboardEventHandler, useContext, useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useNavigate } from 'react-router-dom'
 import { XrplClient } from 'xrpl-client'
-
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why we need this comment even though react-tooltip has been saved as dependency?